source/frontend/src/features/mapSideBar/acquisition/tabs/fileDetails/update/UpdateAcquisitionForm.tsx

```diff
@@ -120,6 +126,30 @@ const AcquisitionDetailSubForm: React.FC<{
     }
   }, [fileStatusTypeCode, setFieldValue]);
 
+  React.useEffect(() => {
```

Comment:
you could keep this useeffect but instead of setting a variable use this to clear the primary contact when the ownerSolicitorContact changes.